plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/merge/MergePresenter.java

```diff
@@ -255,32 +284,32 @@ public SubversionItemNode(SubversionItem dataItem) {
 
             return service.info(project.getLocation(), getData().getURL(), "HEAD", true)
                           .then(new Function<InfoResponse, List<Node>>() {
-                                       @Override
-                                       public List<Node> apply(InfoResponse response) throws FunctionException {
-                                           if (response.getErrorOutput() != null && !response.getErrorOutput().isEmpty()) {
-                                               printErrors(response.getErrorOutput(), constants.commandInfo());
-                                               notificationManager.notify("Unable to execute subversion command", FAIL, FLOAT_MODE);
-                                               return Collections.emptyList();
-                                           }
-
-                                           List<Node> children = new ArrayList<>();
-                                           if (response.getItems().size() > 1) {
-                                               for (int i = 1; i < response.getItems().size(); i++) {
-                                                   SubversionItem item = response.getItems().get(i);
-                                                   if (!"file".equals(item.getNodeKind())) {
-                                                       children.add(new SubversionItemNode(item));
-                                                   }
-                                               }
-                                           }
-
-                                           return children;
-                                       }
-                                   }).catchError(new Operation<PromiseError>() {
-                @Override
-                public void apply(PromiseError error) throws OperationException {
-                    notificationManager.notify(error.getMessage(), FAIL, FLOAT_MODE);
-                }
-            });
+                              @Override
+                              public List<Node> apply(InfoResponse response) throws FunctionException {
+                                  if (response.getErrorOutput() != null && !response.getErrorOutput().isEmpty()) {
+                                      printErrors(response.getErrorOutput(), constants.commandInfo());
+                                      notificationManager.notify("Unable to execute subversion command", FAIL, FLOAT_MODE);
+                                      return Collections.emptyList();
+                                  }
+
+                                  List<Node> children = new ArrayList<>();
+                                  if (response.getItems().size() > 1) {
+                                      for (int i = 1; i < response.getItems().size(); i++) {
```

Comment:
it seems, that these changes are not related to his PR, it looks like formatting issue.